src/dendrogram.jl

```diff
@@ -17,22 +18,39 @@ function treepositions(hc::Hclust, useheight::Bool)
         xs[:, i] .= [x1, x1, x2, x2]
         ys[:, i] .= [y1, ypos, ypos, y2]
     end
-
-    return xs, ys
+    if orientation == :h
+        return ys, xs
+    else
+        return xs, ys
+    end
 end
 
-@recipe function f(hc::Hclust; useheight=true)
+@recipe function f(hc::Hclust; useheight=true, orientation=:v )
```

Comment:
how does this interact with the `orientation` keyword that already exists?

Reply:
This I can't answer. I hope there is no side effect.

Reply:
Does this flip the parabola or not?
```
plot([hc_row, (1:5).^2],yticks=false,xrotation=90,orientation=:horizontal)
```

Reply:
No, it's just not working:
```
julia> plot([hc_row, (1:5).^2],yticks=false,xrotation=90,orientation=:horizontal)
ERROR: Cannot convert Hclust{Float64} to series data for plotting
```
This behaviour isn't changed. This doesn't work in current StatsPlots neither.

Reply:
hmm.. can't think of a good testcase here, I guess its fine. @dashw, you got objections?

Reply:
I think it's is very specific for dendrograms and should not radiate outsides.